src/Stratis.Features.FederatedPeg/FederationGatewaySettings.cs

```diff
@@ -113,7 +112,10 @@ public FederationGatewaySettings(NodeSettings nodeSettings, FederatedPegOptions
         public int MultiSigN { get; }
 
         /// <inheritdoc/>
-        public Money TransactionFee { get; }
+        public Money TransactionFee(int numInputs)
+        {
+            return DefaultTransactionFee + numInputs * InputsTransactionFee;
```

Comment:
Currently `Money.Coins(0.0m)` I this correct?

Reply:
Yes, Until we decide to make this something else.